### DISCUSSION TITLE
Why isn't the code for 1.9.0 available?

### DISCUSSION THREAD

**Discussion by admiralspeedy** [hanai3Bi/Switch-OC-Suite#47] [General]
It appears that the overlay code is still 1.8.3 not 1.9.0 (or 2.0.0-oc-1, according the to the one included in the release package).

Will it be pushed soon?

**Comment by hanai3Bi** [comment]
new sys-clk-oc will not be opensourced.

you can still compile old sys-clk-oc sources(you will have to fix temperature and battery though),
or you can just compile vanilla sys-clk and use it. since difference is not that much right now

**Reply by admiralspeedy** [reply]
You should probably make that clear in the read me then because it's rather confusing.

Also seems pretty shitty to take someone's work, make a few tweaks and keep is closed source for literally no reason.

**Reply by hanai3Bi** [reply]
i will update readme later.

well i don't mind open sourcing sys-clk-oc, but because of some psychopath who abuses my project (not referring to you), i am not gonna open source it unless it is necessary.
like you just said, it is just few tweaks to sys-clk, so go ahead and use sys-clk

**Reply by hanai3Bi** [reply]
also mind you that sys-clk authors explicitly said on their beerware license, they don't care whatever people do with the code.

well of course opensourcing it will be good, but i have my reasonss for it
